CHANGELOG.md

```diff
@@ -12,6 +12,17 @@
 
 ---
 
+[//]: # (START/v1.4.2)
+# v1.4.2
+
+## Fixes
+ * Field of type 'DATE' updates item even if there were no changes. {#137}
+
+## Security
+ * Update dependencies with security vulnerabilities to patched versions. {#144, #145}
```

Comment:
`[non-blocking]`

While it's ok to reference the version bumps that were required, I see that #144 also solved #145. In my opinion I don't think it's worth putting #145 in the changelogs for the following reasons:
- It's solution is already achieved with another PR
- It doesn't look good to link a PR that hasn't been merged.

Reply:
removed #145 from there.